libkbfs/keybase_service_base.go

```diff
@@ -1040,6 +1041,22 @@ func (k *KeybaseServiceBase) TeamDeleted(ctx context.Context,
 	return nil
 }
 
+// StartMigration implements keybase1.ImplicitTeamMigrationInterface for
+// KeybaseServiceBase.
+func (k *KeybaseServiceBase) StartMigration(ctx context.Context,
+	folder keybase1.Folder) (err error) {
+	mdServer := k.config.MDServer()
+	if mdServer == nil {
+		return errors.New("no mdserver")
+	}
+	fav := NewFavoriteFromFolder(folder)
```

Comment:
Why make a favorite?  You can just use `folder.Name` and `folder.Type` in the call below directly.

Reply:
It's to reuse `keybase1.FolderType`->`tlf.Type` conversion. Would you prefer if I do a switch here?

Reply:
Ah I see. Ok.  I guess that makes sense.  Can you please add a comment about it though?